### PR TITLE
feat: Add shorthand cat alias c

### PR DESCRIPTION
**Pull request by allank-fd**
# Fixes # 

### Checklist
- [x] I have made a change to this repository, be it functionality, testing, documentation, spelling, or grammar.
- [x] I updated my branch with the master branch.
- [x] I have added the necessary testing to prove my fix is effective/my feature works (or I did not modify functionality).
- [x] I have added necessary documentation about the functionality in an appropriate .md file.
- [x] I have appropriately commented any code I have modified

### Short description of what this PR does:
- Added shorthand alias for cat `c`
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Renames a given note to destination or moves the note to directory. Name can be
 Removes the given note if it exists. If `-r` or `--recursive` is given, deletes the folders/notes recursively.
 
 ### `notes cat <note-name>`
-
+Shorthand alias also available with `notes c`.
```
Comment (pimterry) [inline README.md]: I think it'd be better to put this after the description of the command (i.e. as a second sentence in the paragraph below. Makes more sense that way, and it's consistent with the rest of the documentation here.

**Review by pimterry — CHANGES_REQUESTED** [review]
It looks like you've reformatted and tweaked lots of unrelated code here, in both the tests and the core script. Can you remove all the changes that aren't required for this PR please, and squash the result back to a single commit? It's very useful to keep commits & PRs to a single set of changes.

I'm open to other changes too, but if you're doing that then it'd be helpful if you could open a separate PR for them, and provide some background on what you're trying to do there & why.

**Comment by allank-fd** [comment]
@pimterry Thanks for the feedback, I've updated the PR to remove the formatting changes. Plus the small change to the Readme to keep it inline with the other command docs.

**Review by pimterry — APPROVED** [review]

**Comment by pimterry** [comment]
Nice work, thanks! Merged :+1:

**Pull request merged**